lib/mockbuilder.py

```diff
@@ -195,12 +203,32 @@ def copy_results(self, package):
             self.timestamp, package.name)
         self._copy_rpms(package.build_cache_dir, package_build_results_dir)
 
+    def create_repository(self):
+        """
+        Create yum repository in build results directory.
+        """
+        result_dir = CONF.get('common').get('result_dir')
+        build_results_dir = os.path.join(
+            result_dir, 'packages', self.timestamp)
+        utils.run_command("createrepo %s" % build_results_dir)
+
+        repo_files_dir = os.path.join(result_dir, "repository_files")
```

Comment:
The actual repository is saved to `result/packages/<timestamp>`
The `result/repository_files` directory is supposed to keep only the `.repo` files, to ease the utilization of the repositories.
Another option would be to add another level to the former directory, i.e. `result/packages/<timestamp>/repository`, and putting the `.repo` file in `result/packages/<timestamp>`. But I prefer to keep them separated.